### PR TITLE
💡 Extend and update documentation for LHCI

### PR DESCRIPTION
**Pull request by tomashejatko**
Related: #52714

### CODE REVIEW

**Comment by tomashejatko** [comment]
Remote aka lighthouse pipeline documentation is missing

File: pages/project/LighthouseSetup.md [hunk 1]
```diff
@@ -65,15 +133,23 @@ Run wizard at you project root directory with this command:
 lhci wizard
 ```
 
-Then set the name your project. Please be specific and do not make a mistake here!!! Lhci server has no chance to edit it when its once created for now.
+Then set the name your project. Please use format *$repo-$mode*, so if you create LHCI project for pipeline (thus local) tests for `milacci-web` project, it will be:
```
Comment (horaklukas) [inline pages/project/LighthouseSetup.md:136]: ...the name of your project.

File: pages/project/LighthouseSetup.md [hunk 2]
```diff
@@ -92,16 +168,39 @@ npm install @lhci/cli --save-dev
...
+You must specify URL against which LHCI will run. You can do this on per branch basis in `ci-branch-config` folder.
 
-## Update Jenkinsfile
-Add this parameter into your *Jenkinsfile* file next to other *run* parameters in your project root directory.
+So, for example if you have basic auth on development branch (and all you feature branches are created from development) and no basic auth on stage:
+
+common.env:
 ```bash
-runLighthouse = true
+LHCI_URL=http://basic_user:basic_password@localhost:3000
```
Comment (horaklukas) [inline pages/project/LighthouseSetup.md:198]: Maybe we should add a note about the password complexity - it must not contain special chars, eg. `-` , `?`, etc..

**Review by horaklukas — APPROVED** [review]
Good job 👏

File: pages/project/LighthouseSetup.md (hunk 1 above)
Comment (tomashejatko) [reply pages/project/LighthouseSetup.md:136]: yes. fixed :)

File: pages/project/LighthouseSetup.md (hunk 2 above)
Comment (tomashejatko) [reply pages/project/LighthouseSetup.md:198]: Good idea!

**Pull request merged**